src/components/ButtonMenu/index.js

```diff
@@ -37,31 +44,38 @@ export default function ButtonMenu(props) {
             onClick={onClick}
             onFocus={onFocus}
             onBlur={onBlur}
-            trigger={ButtonIcon}
+            trigger={trigger}
         >
             {children}
         </PrimitiveMenu>
     );
 }
 
 ButtonMenu.propTypes = {
+    /** The text to be displayed inside the button. */
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
     /** The icon to show if it is passed.
-     * It must be a svg icon or a font icon. It is a required value. */
+     * It must be a svg icon or a font icon. */
     icon: PropTypes.node,
+    /** Describes the position of the icon with respect to label. Options include left and right.
+     * This value defaults to left. */
+    iconPosition: PropTypes.oneOf(['left', 'right']),
     /** The content of the ButtonMenu. Used to render the menuItem elements
      * when the ButtonMenu is open. */
     children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.object]),
-    /** The variant changes the look of the button. Accepted variants include base,
-     * brand, success, border, border-filled, bare-inverse, and inverse.
-     * This value defaults to border-filled. */
+    /** The variant changes the look of the button.
+     * Accepted variants include base, brand, success, destructive, neutral, outline-brand, border, border-filled, inverse and border-inverse. */
```

Comment:
Let's add ``This value defaults to border-filled.`` to the end of the description.